controlpanel/api/serializers.py

```diff
@@ -13,17 +13,10 @@
     S3Bucket,
     User,
     UserApp,
-    UserS3Bucket,
+    UserS3Bucket
```

Comment:
Is black doing this, because it's a bad idea

Reply:
Not sure, I will undo it though